english/contemplate/contemplate.md

```diff
@@ -0,0 +1 @@
+As you sit to consider your finite role in an incomprehensible ever-expanding cosmos, you completely lose touch with reality, transcend your physical form, and traverse dimensions heretofor unknown.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.